src/debug/TestResultsAPITest.ts

```diff
@@ -27,7 +27,7 @@ function initialize() {
 }
 
 initialize();
-TestResultsBuilder.getTestsResultsByBuildId(api, projectName, buildId, 'serverId', 'jobId', new LogUtils('debug')).then(res => {
+TestResultsBuilder.getTestsResultsByBuildId(api, projectName, buildId, 'serverId', 'jobId', "somePath", new LogUtils('debug')).then(res => {
```

Comment:
somepath is not ok. Maybe make it work with proper path and modify the file to actually test the case you added?